railgun/internal/proxy/clientgo_cached_proxy_resolver_test.go

```diff
@@ -108,7 +110,7 @@ func TestCaching(t *testing.T) {
 	assert.NotNil(t, proxy.cache)
 
 	t.Log("intentionally freezing async updates to inspect cache state during tests")
-	proxy.syncTicker.Reset(time.Minute * 1)
+	proxy.syncTicker.Reset(time.Minute * 3)
```

Comment:
note: this timing could really be arbitrary, the reason I increased this was to provide _theoretical_ more cushion for the tests to pass under a system with extreme load. I did this out of a sense that we really don't have control over the kernel schedulers on the Github Actions boxes, but I understand that in practice this may have a `nil` effect.